lib/generators/clearance/specs/templates/features/clearance/visitor_resets_password_spec.rb.tt

```diff
@@ -3,6 +3,15 @@ require "support/features/clearance_helpers"
 
 RSpec.feature "Visitor resets password" do
   before { ActionMailer::Base.deliveries.clear }
+<% if defined?(ActiveJob) -%>
+
+  around do |example|
+    original_adapter = ActiveJob::Base.queue_adapter
+    ActiveJob::Base.queue_adapter = :inline
+    example.run
+    ActiveJob::Base.queue_adapter = original_adapter
```

Comment:
I added `raise Exception` to one of the example and added logging of the queue adapter before we run the example and after we reset the adapter. Everything executes as expected. I don't think `ensure` is necessary here.